client/rest-high-level/src/main/java/org/elasticsearch/client/eql/EqlSearchResponse.java

```diff
@@ -292,123 +292,41 @@ public boolean equals(Object o) {
-                    float percent = (float) args[i];
-                    return new EqlSearchResponse.Count(count, joinKeys, percent);
-                });
-
-        static {
-            PARSER.declareInt(ConstructingObjectParser.constructorArg(), COUNT);
-            PARSER.declareFieldArray(constructorArg(), (p, c) -> XContentParserUtils.parseFieldsValue(p), KEYS,
-                ObjectParser.ValueType.VALUE_ARRAY);
-            PARSER.declareFloat(ConstructingObjectParser.constructorArg(), PERCENT);
-        }
-
-        public Count(int count, List<Object> keys, float percent) {
-            this.count = count;
-            this.keys = keys == null ? Collections.emptyList() : keys;
-            this.percent = percent;
-        }
-
-        public static Count fromXContent(XContentParser parser) {
-            return PARSER.apply(parser, null);
-        }
-
-        public int count() {
-            return count;
-        }
-
-        public List<Object> keys() {
-            return keys;
-        }
-
-        public float percent() {
-            return percent;
-        }
-
-        @Override
-        public int hashCode() {
-            return Objects.hash(count, keys, percent);
-        }
-
-        @Override
-        public boolean equals(Object o) {
-            if (this == o) {
-                return true;
-            }
-            if (o == null || getClass() != o.getClass()) {
-                return false;
-            }
-            Count that = (Count) o;
-            return Objects.equals(count, that.count)
-                && Objects.equals(keys, that.keys)
-                && Objects.equals(percent, that.percent);
-        }
-    }
-
     // Hits
     public static class Hits {
-        public static final Hits EMPTY = new Hits(null, null, null, null);
+        public static final Hits EMPTY = new Hits(null, null, null);
 
         private final List<Event> events;
         private final List<Sequence> sequences;
-        private final List<Count> counts;
         private final TotalHits totalHits;
 
         private static final class Fields {
             static final String TOTAL = "total";
             static final String EVENTS = "events";
             static final String SEQUENCES = "sequences";
-            static final String COUNTS = "counts";
         }
 
-        public Hits(@Nullable List<Event> events, @Nullable List<Sequence> sequences, @Nullable List<Count> counts,
-                    @Nullable TotalHits totalHits) {
+        public Hits(@Nullable List<Event> events, @Nullable List<Sequence> sequences, @Nullable TotalHits totalHits) {
             this.events = events;
             this.sequences = sequences;
-            this.counts = counts;
             this.totalHits = totalHits;
         }
 
         private static final ConstructingObjectParser<EqlSearchResponse.Hits, Void> PARSER =
-            new ConstructingObjectParser<>("eql/search_response_count", true,
```

Comment:
The name here looks like a mistake (left-over from count I presume) so I've changed it to hits instead to preserve the current naming conventions.